internal/kibana/savedobjects.go

```diff
@@ -112,6 +110,50 @@ func (c *Client) findDashboardsNextPage(ctx context.Context, page int) (*savedOb
 	return &r, nil
 }
 
+// FindServerlessDashboards method returns dashboards available in the Kibana instance.
+func (c *Client) FindServerlessDashboards(ctx context.Context) (DashboardSavedObjects, error) {
+	logger.Debug("Find dashboards using the Export Objects API")
+	var foundObjects DashboardSavedObjects
+	allDashboards, err := c.exportAllDashboards(ctx)
```

Comment:
Nit. Not sure if we need this `exportAllDashboards` method. From here you need to go to its implementation to see what it is doing, it is not just exporting all dashboards.
```suggestion
	exportRequest := ExportSavedObjectsRequest{
		ExcludeExportDetails:  true,
		IncludeReferencesDeep: false,
		Type:                  "dashboard",
	}

	return c.ExportSavedObjects(ctx, request)
```

The comment in `exportAllDashboards` is good though. Maybe its part about the limit on exported dashboards could be moved or copied to the description of `FindServerlessDashboards`, as the limit also applies to this method and is the public one.